src/zitadel.rs

```diff
@@ -409,10 +417,15 @@ impl From<User> for ImportHumanUserRequest {
 				email: user.email.clone(),
 				is_email_verified: !user.needs_email_verification,
 			}),
-			phone: Some(Phone {
-				phone: user.phone.clone(),
-				is_phone_verified: !user.needs_phone_verification,
-			}),
+			phone: user.phone.as_ref().map_or_else(
+				|| None,
+				|phone| {
+					Some(Phone {
+						phone: phone.to_owned(),
+						is_phone_verified: !user.needs_phone_verification,
+					})
+				},
+			),
```

Comment:
Isn't it the same as
```rust
			phone: user.phone.as_ref().map(|phone| Phone {
				phone: phone.to_owned(),
				is_phone_verified: !user.needs_phone_verification,
			}),
```
?

Reply:
Yeah, of course it is x) Why did I do that again?